libs/langchain-google-vertexai-web/src/llms.ts

```diff
@@ -0,0 +1,19 @@
+import { type GoogleLLMInput, GoogleLLM } from "@langchain/google-webauth";
+
+/**
+ * Input to LLM model class.
+ */
+export interface VertexAIInput extends GoogleLLMInput {}
+
+/**
+ * Integration with a LLM model.
+ */
+export class VertexAI extends GoogleLLM {
+  static lc_name() {
+    return "VertexAI";
+  }
+
+  constructor(fields?: VertexAIInput) {
+    super(fields);
```

Comment:
I'm wondering if we should pass the `platformType` field explicitly setting it to "gcp" (here and in the other constructors).